index.bs

```diff
@@ -10071,7 +10036,8 @@ in the node.
 				while connected inputs and/or script references exist. Such
 				nodes SHOULD return <code>false</code> from
 				{{process()}} which allows the presence or absence of
-				connected inputs to determine whether active processing occurs.
+				connected inputs to determine whether the {{AudioWorkletNode}}s
```

Comment:
Maybe replace "AudioWorkletNodes actively processing occurs" -> "AudioWorkletNode is actively processing"?